dailyperf/tests/icon_border/test.md

```diff
@@ -2,9 +2,9 @@
 
 ![Border VS Icons](img.png)
 
-_Between border and icon, witch one impact the most the performance?_
+*Between a border and an icon, which one impacts performance the most?*
```

Comment:
When you have to choose between 2 visuals: border color or icon, which one impacts performance the most?